### PR TITLE
Ability to provide a lookup table to fetch the remote address.

### PR DESCRIPTION
**Pull request by amandiobm**
Related to issue #8 

@jamesmills  & @ridz1208 I did some initial implementation on that issue, please take a look and feel free to add comments, commits, etc. ;)

I initially thought of creating a lookup table so we can search in whatever attribute inside the request helper, by any key. Having in mind when the key is found inside the attribute, that key will be used.

### CODE REVIEW

**Comment by jamesmills** [comment]
Hi @amandiobm!

Thank you for this suggested solution to the initial issue (https://github.com/jamesmills/laravel-timezone/issues/8) which @ridz1208 submitted. 

This solution is much more thorough than the one I was going to suggest so thank you for this and awsome work.

I don't see anything that would need changing for it to be merged.

Once you have updated the README I would be happy to merge, tag and push this to the world!

Thank you.

**Comment by amandiobm** [comment]
Neat. I still need to update the README, the config description, a double-check, and we're all good ;)

It would be nice to hear from @ridz1208 if that approach would help him.

See you soon, and is always a pleasure ;)

**Comment by amandiobm** [comment]
I think we're all done @jamesmills . take a look.

File: README.md [hunk 1]
```diff
@@ -118,6 +118,10 @@ By default, the timezone will be overwritten at each login with the current user
 
 By default, the date format will be `jS F Y g:i:a`. To override this configuration, you just need to change the `format` property inside the configuration file `config/timezone.php` for the desired format.
 
+### Lookup Array
+
+this lookup array configuration makes it possible to find the remote address of the user in any attribute inside the Laravel `request` helper, by any key. Having in mind when the key is found inside the attribute, that key will be used. By default, we use the `server` attribute with the key `REMOTE_ADDR`. To override this configuration, you just need to change the `lookup` property inside the configuration file `config/timezone.php` for the desired lookup.
```
Comment (ridz1208) [inline README.md]: This should start with an uppercase letter

Comment (amandiobm) [reply README.md]: Done. Tks.

**Comment by amandiobm** [comment]
Hey @jamesmills any follow-up?

**Comment by jamesmills** [comment]
Hi!

Looks great, I will look at the PR when I am in the office tomorrow and update the changelog and tag a new release. 

Thank you for your continued support!

James

**Comment by jamesmills** [comment]
Done. Thanks again.

**Pull request merged**